README.md

```diff
@@ -217,6 +216,45 @@ You can assign a lambda to a variable, as shown above.
 
 A branch of mathematics that uses functions to create a [universal model of computation](https://en.wikipedia.org/wiki/Lambda_calculus).
 
+This is in contrast to a [Turing machine](https://www.youtube.com/watch?v=dNRDvLACg5Q), an equivalent model.
```

Comment:
Left some thoughts/comments on this in the telegram chat, please check it out and let me know